site/content/docs/tutorials/supervisor-without-concierge-demo.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Learn to use the Pinniped Supervisor without the Concierge
+description: See how the Pinniped Supervisor can work directly with the Kube API server to provide authentication to Kubernetes clusters.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Supervisor without Concierge
+    parent: tutorials
+    weight: 200
+---
+
+## Overview
+
+This tutorial shows how to use the Pinniped Supervisor and Pinniped command-line tool to provide provided federated identity
+with a single sign-on user experience on many Kubernetes clusters, without using the Pinniped Concierge.
+If you would like to learn how to use the Pinniped Supervisor and Concierge together,
+please instead see this other tutorial:
+- [Concierge with Supervisor: a complete example of every step, demonstrated using GKE clusters]({{< ref "concierge-and-supervisor-demo" >}})
+
+The Kubernetes API server can be configured to trust an OIDC identity provider to provide authentication
+for the cluster. This is done by setting the `--oidc-*` command-line flags of the `kube-apiserver` command-line tool inside
+the Pod spec of the Kubernetes API server Pods. The details of these command-line flags are described in the
+[Kubernetes kube-apiserver documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
+These flags can be used to configure the Pinniped Supervisor as the OIDC provider for your clusters.
+
+If your cluster's Kubernetes distribution does not allow you to adjust these command-line flags, then this approach will
+not work for those clusters. For example, most cloud providers will not allow these flags to be adjusted. In that case,
+you can use the Pinniped Concierge to provide equivalent the functionality on those clusters.
+
+Additionally, if you would like to be able to easily change this configuration at any time on your cluster, then
+you can use the Pinniped Concierge instead of these `kube-apiserver` command-line flags, even on a cluster where you
+have control over these flags. The Pinniped Concierge offers a custom resource called `JWTAuthenticator` which can
```

Comment:
to be consistent, don't monospace "JWTAuthenticator"

Reply:
Fixed.